Add EKS cluster create/teardown tasks (#606)
tektoncd/catalog: task/eks-cluster-create/0.1/README.md

# EKS Cluster Create

The Task `eks-cluster-create` can be used to create an EKS cluster in an AWS account, using the `eksctl` command,
and fetch a kubeconfig that can be used (in a context with both kubectl and aws credentials available) to make 
requests to the cluster.

## Parameters

> * **cluster-name**: The name of the EKS cluster which will be teared down. (_required_)

Reviewer: I guess this is what you meant...right?
```suggestion
* **cluster-name**: The name of the EKS cluster which you want to spin. (_required_)
```

Author: fixed in cc49d35

> * **cluster-version**: The EKS version to install. (_default_:1.17

Reviewer: nit:
```suggestion
* **cluster-version**: The EKS version to install. (_default_:1.17)
```

Author: fixed in cc49d35
